inspire/modules/formatter/views.py

```diff
@@ -84,3 +88,36 @@ def get_latex_file(recid, latex_format):
             )
         }
     )
+
+
+@blueprint.route("/export-as/<path:export_format>/<path:ids>",
+                 methods=['GET', 'POST'])
+def download_all_latex(ids, export_format):
```

Comment:
Add here a docstring explaining what are the `ids` and `export_format` parameters and what is the result value.

We use `Sphinx` format. See https://github.com/inspirehep/invenio-workflows/blob/master/invenio_workflows/api.py#L71